tests/WinRTTests.cpp

```diff
@@ -240,8 +240,8 @@ void DoHStringDifferentValueComparisonTest(const wchar_t (&lhs)[LhsSize], const
     DoHStringComparisonTest<InhibitArrayReferences, IgnoreCase>(lhsUniqueStr, rhsUniqueStr, 1);
 
 #ifdef WIL_ENABLE_EXCEPTIONS
-    std::wstring lhsWstr(lhs, 7);

```

Comment:
I can't, for the life of me, understand why this was originally done, however it's obvious UB to do this and the tests have no reliance on it